Makefile

```diff
@@ -54,6 +59,16 @@ test-setup: test/test_playbooks/server_vars.yml
 	pip install --upgrade pip
 	pip install -r requirements-dev.txt
 	pip install -r https://raw.githubusercontent.com/ansible/ansible/devel/requirements.txt 
+ifdef TRAVIS_PULL_REQUEST
+	@echo "Handling dependencies for PR #$(TRAVIS_PULL_REQUEST)"
+	@echo "Repository: $(TRAVIS_REPO_SLUG)"
+	@echo "$(PR_MSG)"
```

Comment:
This appears to be null? https://travis-ci.org/theforeman/foreman-ansible-modules/jobs/462829899#L657

Reply:
It's strange. If i run it locally with `TRAVIS_PULL_REQUEST=206` and `TRAVIS_REPO_SLUG=theforeman/foreman-ansible-modules` it resolves correctly to 203.

Reply:
We could use `TRAVIS_COMMIT_MESSAGE` ?

Reply:
Let's see

Reply:
I guess, that's not it:
```Merge fa53c9e3c602ea470fd29699098868843901919c into 9b39d58591e9e0bebd411fbc66a887bc83c24541```

Now it's getting really strange. Seems like the API sometimes produces the right information.

Reply:
I think that ENV var contains the latest commit message, so it'll mean the author must add the requirement to the commit message. Which IMO is okay?

Reply:
It's the generated Commit Message made by checking out the `refs/pulls/<number>/merge` branch. Anyway, i do not like to have such information in any commit message.